jetty-core/jetty-http2/jetty-http2-client-transport/src/main/java/org/eclipse/jetty/http2/client/transport/HttpClientTransportOverHTTP2.java

```diff
@@ -42,9 +42,10 @@
 import org.eclipse.jetty.util.Promise;
 import org.eclipse.jetty.util.annotation.ManagedAttribute;
 import org.eclipse.jetty.util.annotation.ManagedObject;
+import org.eclipse.jetty.util.thread.Invocable;
 
 @ManagedObject("The HTTP/2 client transport")
-public class HttpClientTransportOverHTTP2 extends AbstractHttpClientTransport
+public class HttpClientTransportOverHTTP2 extends AbstractHttpClientTransport implements Invocable
```

Comment:
The `HttpClientTransport` super-interface already extends from `Invocable`.

Reply:
Done.